### PR TITLE
[ADI][ADMIN] Moderate idea threads and thread comments + Refacto idea moderation

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Controller/Admin/AdminThreadController.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    /**
+     * Moderates a thread.
+     *
+     * @Route("/disable", methods={"GET"}, name="app_admin_thread_disable")
+     */
+    public function disableAction(Thread $thread, ObjectManager $manager): Response
+    {
+        $thread->disable();
+        $manager->flush();
+        $this->addFlash('sonata_flash_success', sprintf('Le commentaire « %s » a été modéré avec succès.', $thread->getId()));
```
Comment (JeremieSamson) [inline src/Controller/Admin/AdminThreadController.php]:
```suggestion
        $this->addFlash('sonata_flash_success', sprintf('Le fil de discussion « %s » a été modéré avec succès.', $thread->getId()));
```

File: src/Controller/Admin/AdminThreadController.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    /**
+     * Enable a thread.
+     *
+     * @Route("/enable", methods={"GET"}, name="app_admin_thread_enable")
+     */
+    public function enableAction(Thread $thread, ObjectManager $manager): Response
+    {
+        $thread->enable();
+        $manager->flush();
+        $this->addFlash('sonata_flash_success', sprintf('Le commentaire « %s » a été activé avec succès.', $thread->getId()));
```
Comment (JeremieSamson) [inline src/Controller/Admin/AdminThreadController.php]:
```suggestion
        $this->addFlash('sonata_flash_success', sprintf('Le fil de discussion « %s » a été activé avec succès.', $thread->getId()));
```

File: src/Admin/IdeasWorkshopIdeaAdmin.php [hunk 3]
```diff
@@ -189,11 +189,8 @@ protected function configureListFields(ListMapper $listMapper)
                     'show_contributors' => [
                         'template' => 'admin/ideas_workshop/idea/list_show_contributors.html.twig',
                     ],
-                    'enabled' => [
-                        'template' => 'admin/ideas_workshop/idea/list_action_enabled.html.twig',
-                    ],
-                    'disabled' => [
-                        'template' => 'admin/ideas_workshop/idea/list_action_disabled.html.twig',
+                    'moderate' => [
+                        'template' => 'admin/ideas_workshop/thread/list_action_moderate.html.twig',
```
Comment (ottaviano) [inline src/Admin/IdeasWorkshopIdeaAdmin.php]: maybe move this template in the `admin/ideas_workshop/` directory?

File: src/Repository/ThreadCommentRepository.php [hunk 4]
```diff
@@ -9,9 +9,21 @@
...
 {
     use AuthorTrait;
+    use UuidEntityRepositoryTrait;
 
     public function __construct(RegistryInterface $registry)
     {
         parent::__construct($registry, ThreadComment::class);
     }
+
+    public function findOneByUuid(string $uuid, bool $disabledEntity = false): ThreadComment
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply src/Repository/ThreadCommentRepository.php]: Done.

File: src/Admin/IdeasWorkshopIdeaAdmin.php (hunk 3 above)
Comment (illusionOfParadise) [reply src/Admin/IdeasWorkshopIdeaAdmin.php]: Done.

**Pull request merged**